web/templates/datasets/dataset.html

```diff
@@ -31,6 +26,19 @@ <h1>{{ dataset.title }}</h1>
                         <dd>{{ dataset.other_external_id | default:'-' }}</dd>
                         <dt>Other comments</dt>
                         <dd>{{ dataset.comments | default:'-' | linebreaks }}</dd>
+                        {% if exposure_list|length > 0 %}
+                        <dt>Endpoints</dt>
```

Comment:
Following this morning discussion, this maybe should be changed to a more meaningful word.

`Data repository` or something.

Reply:
changed to published on